index.rst

```diff
@@ -6,20 +6,423 @@
+
+-	Pfeifer Turbopump (to be used from 1 Torr to ~10−6 Torr), complete with vacuum hose, pressure sensor and the sensor’s electronic display
+
+-	Ion pump (to be used from 10−6 Torr onwards)
+
+.. figure:: /_static/turbo_pump_equipment.png
+   :name: turbo-pump-equipment
+   :alt: Turbo pump equipment
+
+   Pfeifer turbopump and accessories.
+
+The Pfeifer turbopump lives in the AuxTel storage container on Calibration Hill. 
+The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container. 
+The cardboard storage box is labeled. 
+The key to the storage container is found in the key box on the second floor (control room floor) of the main building, to the right of the main entrance doors. 
+Mario Rivera also has a copy. 
+
+.. figure:: /_static/auxtel_storage.png
+   :name: auxtel-storage
+   :alt: Auxtel Storage
+
+   Turbo pump supplies located in the storage unit outside of the AuxTel dome.
+
+The ion pump (mounted on LATISS) doesn’t have enough capacity to evacuate the dewar from normal atmospheric pressure to working pressure, so the Pfeifer Turbopump has to be used first. 
+The ion pump should be started only after ~10−6 Torr is reached (as the high pressure could destroy the ion pump).
+
+Connect the turbopump to the dewar
+-----------------------------------
+
+Put the turbopump in a stable position close enough to the dewar position (to be close to the flange connections). 
+An example setup is shown in Figure 5. 
+
+.. warning::
+   The Pfeifer Turbopump must be supported on a stable surface and strapped to the observing deck grating to ensure its safety in the case of an earthquake. 
+   The turbopump's operation is degraded if it is not operating on a secure surface.
+   A stout stool can be found on the observing floor of AuxTel which is perfect for this setup. 
+   It is likely that the instrument will need to be rotated to +20 degrees to accommodate this. 
+   The telescope mount may also need to be rotated to accommodate all of the power supplies necessary to run the turbopump and vacuum sensors.
+
+.. figure:: /_static/turbopump_setup.png
+   :name: turbopump-setup
+   :alt: Turbopump setup
+
+   Left - Connecting the turbopump to the dewar. Right - Vacuum hose connection to the dewar. 
+
+Remove the vacuum covers, and clean the turbopump and the hose flange with isoporpyl alcohol (using gloves), to avoid introducing contaminants inside the turbopump and hose.
+Connect the hose to the turbopump flange. 
+The hose has to be in a relaxed shape, with no turn or folds. 
+
+After checking the instrument dewar valve is closed, clean both the vacuum hose and instrument dewar flange with alcohol.
+Connect the other end of the hose to the dewar, without forcing the hose into a stressed position.
+This is why it may be necessary to rotate Nasmyth #2 between 0° and 20° to find a comfortable fit. 
+
+Check the turbopump output valve is closed.
```

Comment:
turbopump output valve? I didn't think there was one?

Reply:
There is a vent which has to be closed... which is a black thumbscrew where the pipe enters the red box... but that should never really be opened.

Reply:
That's what it is called. 👍

Reply:
That's what it is called. 👍